creating_packages/package_approaches.rst

```diff
@@ -93,54 +91,44 @@ library, in the same package.
 
 .. note::
 
-    A complete working example of the following code can be found in a github repo. You should be
-    able to run:
+    A complete working example of the following code can be found in the examples repo: https://github.com/conan-io/examples
 
     .. code:: bash
 
-        $ git clone https://github.com/memsharded/hello_multi_config
-        $ cd hello_multi_config
-        $ conan create . user/channel -s build_type=Release
-        $ conan create . user/channel -s build_type=Debug --build=missing
+        $ git clone https://github.com/conan-io/examples.git
+        $ cd features/multi_config
+        $ conan create . user/channel
 
-Creating a multi-configuration Debug/Release package is simple, see the following example
-using CMake.
+Creating a multi-configuration debug/release package is simple
 
-The first step will be to remove ``build_type`` from the settings. It will not be an input setting, the
-generated package will always be the same, containing both Debug and Release artifacts.
-The Visual Studio runtime is different for debug and release (``MDd`` or ``MD``) and is set using the default runtime (MD/MDd). If this meets your needs, we recommend removing the ``runtime`` subsetting in the
-``configure()`` method:
+The first step will be to remove ``build_type`` from the settings. It will not be an input setting and the generated package will always
+contain both debug and release artifacts.
```

Comment:
may we add some clarification, what happens to other configurations besides Release/Debug (e.g. RelWithDebInfo), is it unsupported case or it will just work out of the box?

Reply:
probably it will just work, but I don't want to make the example more complex or make the explanation too long for the user

Reply:
I think it is ok for this PR to keep the Debug/Release as it was before.